<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AeroQuad Configurator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/AQ/AeroQuadMacIcon.ico</normaloff>:/AQ/AeroQuadMacIcon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QToolBox" name="SideMenu">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="tabSpacing">
       <number>6</number>
      </property>
      <widget class="QWidget" name="side_menu_info_page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>178</width>
         <height>199</height>
        </rect>
       </property>
       <attribute name="icon">
        <iconset>
         <normaloff>../resources/info.png</normaloff>../resources/info.png</iconset>
       </attribute>
       <attribute name="label">
        <string>Info</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="side_menu_setting_page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>178</width>
         <height>199</height>
        </rect>
       </property>
       <attribute name="icon">
        <iconset>
         <normaloff>../resources/settings.png</normaloff>../resources/settings.png</iconset>
       </attribute>
       <attribute name="label">
        <string>Settings</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="side_menu_troubleshooting_page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>178</width>
         <height>199</height>
        </rect>
       </property>
       <attribute name="icon">
        <iconset>
         <normaloff>../resources/graphic.png</normaloff>../resources/graphic.png</iconset>
       </attribute>
       <attribute name="label">
        <string>Data plot</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="side_menu_mission_planer_page">
       <attribute name="label">
        <string>Page</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="subPanel"/>
    </item>
    <item row="1" column="1">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="status">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Not connected to AeroQuad</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comPort">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="baudRate">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonConnect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonDisconnect">
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="button_home">
      <property name="text">
       <string>Home</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../resources/home.png</normaloff>../resources/home.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>line</zorder>
   <zorder>SideMenu</zorder>
   <zorder>subPanel</zorder>
   <zorder>button_home</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSettings">
     <property name="title">
      <string>Settings</string>
     </property>
     <addaction name="actionOpenSettings"/>
     <addaction name="actionSaveSettings"/>
    </widget>
    <widget class="QMenu" name="menuCalibrations">
     <property name="title">
      <string>Calibrations</string>
     </property>
     <addaction name="actionOpenCalibrations"/>
     <addaction name="actionSaveCalibrations"/>
    </widget>
    <widget class="QMenu" name="menuPreferences">
     <property name="title">
      <string>Preferences</string>
     </property>
     <addaction name="actionBootUpDelay"/>
     <addaction name="actionCommTimeout"/>
    </widget>
    <addaction name="menuSettings"/>
    <addaction name="menuCalibrations"/>
    <addaction name="menuPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpenSettings">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSaveSettings">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionOpenCalibrations">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSaveCalibrations">
   <property name="text">
    <string>Save....</string>
   </property>
  </action>
  <action name="actionBootUpDelay">
   <property name="text">
    <string>Boot Up Delay...</string>
   </property>
  </action>
  <action name="actionCommTimeout">
   <property name="text">
    <string>Comm Timeout...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="AQresources.qrc"/>
 </resources>
 <connections/>
</ui>
